docs/release-notes/NuGet-6.1.md

```diff
@@ -0,0 +1,120 @@
+
+* Async Service Provider should be invoked on threadpool threads via the built in TService,TInterface extensions instead of custom casting - [#11200](https://github.com/NuGet/Home/issues/11200)
+
+* [Bug]: VS crashes when package sources dropdown in PM UI has multiple package sources whose names are equal when compared using Culture Ignore Case - [#11241](https://github.com/NuGet/Home/issues/11241)
+
+* The strings "ms"  and "sec" are not localized on Output - Package Manager window. - [#11297](https://github.com/NuGet/Home/issues/11297)
+
+* Sort Package Source dropdown list using locale/culture setttings in PM UI - [#11298](https://github.com/NuGet/Home/issues/11298)
+
+* Update SDPX license list from da7ecca to fafa781 - [#11309](https://github.com/NuGet/Home/issues/11309)
+
+* [Bug]: NuGet.Localization isn't generated - [#11328](https://github.com/NuGet/Home/issues/11328)
+
+* [Bug]:  assembly-loading MEF composition in NuGet.Tools VS package's synchronous event handlers can cause responsiveness delays - [#11334](https://github.com/NuGet/Home/issues/11334)
+
+* [Bug]: NuGet SDK resolver should not throw if project path is NULL - [#11376](https://github.com/NuGet/Home/issues/11376)
+
+* [Bug]: PackageSourceMapping inconsistencies should be reported - [#11385](https://github.com/NuGet/Home/issues/11385)
+
+* [Bug]: `RegistryKeyUtility.GetValueFromRegistryKey()` has unused parameter, leading to incorrect results - [#11407](https://github.com/NuGet/Home/issues/11407)
+
+* Reduce heap allocations in ResolverUtility.FindLibraryByVersionAsync - [#11409](https://github.com/NuGet/Home/issues/11409)
+
+* [Bug]: Package Source Mapping matches found should not be logged for projects - [#11413](https://github.com/NuGet/Home/issues/11413)
+
+* [Bug]: Avoid explicitly casting the result of GetService/GetServiceAsync, use the extension methods instead - [#11451](https://github.com/NuGet/Home/issues/11451)
+
+* [Bug]: Package Source options in VS does not announce package sources or checkbox status - [#11482](https://github.com/NuGet/Home/issues/11482)
+
+* [Bug]: VS package source options clears all checkboxes when adding or removing sources - [#11521](https://github.com/NuGet/Home/issues/11521)
+
+**StillOpens:**
+
+* Remove project.json pack - [#7931](https://github.com/NuGet/Home/issues/7931)
+
+* Async implementation of IVS Package Installer/Uninstaller services - [#8896](https://github.com/NuGet/Home/issues/8896)
+
+**Nones:**
+
+* NuGet no longer uses CPS's JTF - [#8317](https://github.com/NuGet/Home/issues/8317)
+
+* [5.x] Make restore types that are project.json-specific obsolete - [#9149](https://github.com/NuGet/Home/issues/9149)
+
+* [Bug]:  Use IAsyncServiceProvider extension methods instead of type casting ourselves - [#11339](https://github.com/NuGet/Home/issues/11339)
+
+* Remove NUGET_FRAMEWORKS_INTERNAL compiler constant - [#11418](https://github.com/NuGet/Home/issues/11418)
```

Comment:
Removing this issue from the release notes because the issue has `Type:Engineering` label.